includes/dbFacile.php

```diff
@@ -24,10 +24,10 @@
 function dbQuery($sql, $parameters = array()) {
 	global $fullSql, $debug;
 	$fullSql = dbMakeQuery($sql, $parameters);
-        if($debug) { 
-          print Console_Color::convert("\nSQL[%y".$fullSql."%n] ");
-          #echo("\nSQL[".$fullSql."] "); 
-        }
+        //if($debug) { 
```

Comment:
whoops!  I was trying to do debug on just SQL errors.  Might be a good idea to output SQL errors to the log file or a separate log file.